### PR TITLE
faq for add/remove subgroup and install extension from whl file

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

File: doc/faq.md [hunk 1]
```diff
@@ -7,9 +7,55 @@ Configuration can be put in either *xxx.cli.md* or *xxx.az.md* in *Azure/azure-r
...
+## what would the clear-output-folder clear
+* Instead of using --output-folder in the command line to specify which folder you want the extension to be generated. we use --azure-cli-extension-folder=path-to-local-azure-cli-extensions-repo now. 
+* the clear-output-folder would clear everything under path-to-local-azure-cli-extensions-repo/src/extension-name except the manual folder which is path-to-local-azure-cli-extensions-repo/src/extension-name/azext_extension_name/manual now.
+
+
+## How to add/remove subgroup 
+* we can add or remove subgroup by using directive
+* Let's say we want to
+* Remove subgroup:
+  > datashare share -> datashare
```
Comment (RodgeFu) [inline doc/faq.md:43]: it's better to have a full command example. people who dont know what datashare is may be lost here.

Comment (qiaozha) [reply doc/faq.md]: changed. please check again

File: doc/faq.md [hunk 2]
```diff
@@ -7,9 +7,55 @@ Configuration can be put in either *xxx.cli.md* or *xxx.az.md* in *Azure/azure-r
...
+## what would the clear-output-folder clear
+* Instead of using --output-folder in the command line to specify which folder you want the extension to be generated. we use --azure-cli-extension-folder=path-to-local-azure-cli-extensions-repo now. 
+* the clear-output-folder would clear everything under path-to-local-azure-cli-extensions-repo/src/extension-name except the manual folder which is path-to-local-azure-cli-extensions-repo/src/extension-name/azext_extension_name/manual now.
+
+
+## How to add/remove subgroup 
+* we can add or remove subgroup by using directive
+* Let's say we want to
+* Remove subgroup:
+  > az datashare share -> az datashare
```
Comment (RodgeFu) [inline doc/faq.md:43]: please also add the arguments:
az datashare share --argument value ... -> az datashare --argument value ...

Comment (qiaozha) [reply doc/faq.md]: changed. please review again.

**Pull request merged**